pkg/v1/webhooks/certificates.go

```diff
@@ -122,3 +135,30 @@ func ValidateTLSSecret(tlsSecret *corev1.Secret, certGracePeriod time.Duration)
 	}
 	return nil
 }
+
+// Write secret to persistent storage
+func updatePersistentTLSSecret(ctx context.Context, clusterClient client.Client, tlsSecret *corev1.Secret) error {
```

Comment:
Rename to updateTLSsecret. Nothing special about saving to k8s.

Reply:
I guess you mean `updateTLSSecret` or maybe `createOrUpdateTLSSecret` (which is more consistent with the naming convention we used in other controllers). But this is just my personal opinion, whatever name makes sense would be feasible.

Reply:
concistency is good

Reply:
done